poetry.lock

```diff
@@ -1,10 +1,15 @@
+# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
```

Comment:
how does this change when passing to Python 3.11?

I would recommend waiting for the 311 pull to be completed and then adapt this pull accordingly.

Reply:
Yes I absolutely agree, with the move to 3.11 we should probably also review our dependencies and update the pinned versions, so that will lead to re-generating the `poetry.lock` if we do.

As `idlbridge` is a bit of a hacky package to install - and is basically abandoned anyway - I'm thinking I'll actually remove it from `pyproject.toml` and for the specific use case someone would want it they can install it manually.